Android.mk

```diff
@@ -26,12 +26,49 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-LOCAL_PATH:= $(call my-dir)
+LOCAL_PATH := $(call my-dir)
```

Comment:
Add default with mdns value for Android.mk?

`WITH_MDNS ?= mojo`

Reply:
We have tests with no WITH_MDNS specified.

Reply:
WITH_MDNS is used by Android.mk and travis test scripts. It has no default value in Android.mk because we didn't support it before. Now we are adding the support, we should set default value to mojo, thus it can be used without any modifications.